### ISSUE TITLE
[temporary solution]How to fix "Purchasing problem"/[временое решение]Как заставить бота работать

### ISSUE THREAD

**Issue by immorro** [Maxx53/SCMBot#549]

**Comment by immorro** [comment]
й

**Comment by immorro** [comment]
ok, im not check this, but i think it will be work for all items in market:
change this:
```
public class ListingInfo
        {
            [JsonProperty("listingid")]
            public string listingid { get; set; }

            [JsonProperty("converted_price")]
            public int price { get; set; }

            [JsonProperty("converted_fee")]
            public int fee { get; set; }

            [JsonProperty("asset")]
            public ItemAsset asset { get; set; }

            [JsonProperty("steamid_lister")]
            public string userId { get; set; }
        }
```
for this
```
public class ListingInfo
        {
            [JsonProperty("listingid")]
            public string listingid { get; set; }

            [JsonProperty("converted_price")]
            public int price { get; set; }

            [JsonProperty("converted_publisher_fee")]
            public int fee { get; set; }

            [JsonProperty("asset")]
            public ItemAsset asset { get; set; }

            [JsonProperty("steamid_lister")]
            public string userId { get; set; }
        }
```

**Comment by Rocky54** [comment]
Где это менять?

**Comment by nikita971** [comment]
поиск знаешь что такое?:/

**Comment by Rocky54** [comment]
@nikita971 поиск по чему? Поиск где?

**Comment by nikita971** [comment]
@immorro ,спасибо за инфу

**Comment by Rocky54** [comment]
@nikita971 
Ответь пожалуйста... Помоги)

**Comment by Rocky54** [comment]
@nikita971 
Я в logfile вписывал - не помогло, видимо не туда

**Comment by nikita971** [comment]
в логфайл вписывал? лучше бы ты это не писал. я в расширении от хрома пока сделал, всё работает

**Comment by Rocky54** [comment]
@nikita971 

Кинь ссылку на расширение, или название)

**Comment by nikita971** [comment]
STH

**Comment by Rocky54** [comment]
@nikita971 
И куда тут вписывать? :D 
![image](https://cloud.githubusercontent.com/assets/12471396/11448325/b111e10a-9566-11e5-8a80-a115d36f9702.png)

**Comment by nikita971** [comment]
неужели ты не понял что это надо в код программы вписывать:/

**Comment by Rocky54** [comment]
@nikita971  
И как? :D

**Comment by Igzibed** [comment]
@Rocky54 качаешь source файлы программы, находишь поиском в коде то что написано выше,меняешь код, компилируешь, profit =)

**Comment by nikita971** [comment]
@Igzibed, у него наверное мозг кипит от того что ты написал:D

**Comment by mrmikas555** [comment]
Как оно вообще помогает? Объясните пожалуйста...

**Comment by Rocky54** [comment]
@nikita971 
ОМГ, а ты как родился сразу всё знал? Или тебя тоже кто то научил?

**Comment by Rocky54** [comment]
@Igzibed 

Сделал всё, спасибо. А как теперь это всё открыть?)

**Comment by Rocky54** [comment]
@Igzibed 
Вернее переобразовать в экзешник

**Comment by rnd0m** [comment]
@Rocky54, ты пропустил самый главный шаг - "компилируешь"

**Comment by rnd0m** [comment]
@Igzibed, о, привет. У тебя такой же аватар в стиме? Помню в теме smb тебя очень ругали )

**Comment by Rocky54** [comment]
@rnd0m

И как мне выполнить этот шаг?)

**Comment by nikita971** [comment]
@Rocky54 как это не печально, сам всему научился:/

**Comment by Rocky54** [comment]
@nikita971 
Ну значит будь добр, помоги другим научиться

**Comment by nikita971** [comment]
@Rocky54 ты вообще понимаешь о чем тут говорят или намекают?))) что нужно сделать и где надо тебе только догадаться)

**Comment by Rocky54** [comment]
@nikita971 

Я всё уже сделал))) Мне осталось только всё это в экзешник превратить (скомпилировать), вопрос как?

**Comment by Rocky54** [comment]
Почему так трудно сказать?

**Comment by Igzibed** [comment]
@rnd0m привет, да в steam такой же автар и ник, но это только один из моих аккаунтов.

**Comment by Rocky54** [comment]
@immorro 
Увы.... Но всё равно ошибка 
![image](https://cloud.githubusercontent.com/assets/12471396/11451639/f70a6c3c-95dd-11e5-8283-fd44702e4b7e.png)

**Comment by immorro** [comment]
@Rocky54 , эта ошибка, потому что кто-то купил лот быстрее бота. Если бы не работало - было бы что-то вроде этого:
![Image](https://cloud.githubusercontent.com/assets/12471396/11424239/7997d596-945c-11e5-9ca6-e4572c241874.png)

**Comment by qwarzen** [comment]
Ой спасибо большое

**Comment by Rocky54** [comment]
@immorro 
Ты не понял, а вернее я не так написал. Я знаю что это за ошибка, но она теперь даётся раз в минуту - две и именно по одинаковому кол-ву. То есть сейчас подалось 3 одновременно, то через минуту опять 3 будет. Что то мне подсказывает это явно не из за того что кто то купил быстрее

**Comment by Rocky54** [comment]
@immorro 
Поэтому покупок так же нету как и без всех этих действий :(

**Comment by Rocky54** [comment]
@immorro 

Даже поставил нереально крутую задержку и кол - во запросов но увы... Всё равно

**Comment by Rocky54** [comment]
@immorro 

О, теперь ещё и такое 
![image](https://cloud.githubusercontent.com/assets/12471396/11452544/2ea5e158-95fc-11e5-80d7-b5c8064e1062.png)

**Comment by Rocky54** [comment]
@immorro 

О, теперь ещё и такое 
![image](https://cloud.githubusercontent.com/assets/12471396/11452544/2ea5e158-95fc-11e5-80d7-b5c8064e1062.png)

**Comment by immorro** [comment]
@Rocky54 , хз почему так...у меня все норм работало. Может собрал неправильно, попробуй эту:
https://drive.google.com/open?id=0B_vJ3_En9BJyRVFJNld1ZFlqM2c

**Comment by Rocky54** [comment]
@immorro 

А ошибка что мол не успел купить у тебя часто?

**Comment by immorro** [comment]
да я не пользую этот бот почти, прочекал, что покупка прошла и все...

**Comment by rnd0m** [comment]
@Rocky54, попробуй стандартную проверку пройти - дешевый предмет поставь по цене, по которой так и так продается

**Comment by Dragonofhell** [comment]
баны не дают уже?

**Comment by kashakden** [comment]
@Igzibed омг, как же ты в своё время достал демпингом цен на редлайны.

**Comment by kashakden** [comment]
@immorro а чем пользуешься если не секрет?

**Comment by immorro** [comment]
@kashakden , своими собственными наработками

**Comment by kashakden** [comment]
@immorro звучит загадочно =\
Ну а всё же, используешь какой-то самописный софт или готовый, но доработанный своими ручками? Если готовый, то какой?

**Comment by immorro** [comment]
@kashakden , да, сам написал

**Comment by greenberg3** [comment]
Я тут решил компильнуть, открываю билд, и тупо грузит (не открывается программа) в процессах ее нет. в визуалке открывается. что за хрень?

**Comment by stranger2001** [comment]
кривые руки у пацана, вот шо за хрень:DDD

**Comment by greenberg3** [comment]
ранее компилировал и все нормально было.

**Comment by Yakman3** [comment]
um, i don't speak russian but would love to know how to put in that code, any suggestions?

**Comment by immorro** [comment]
@Yakman3 u can download compiled exe from comment above or:
1 download visual studio
2 download scm source files
3 change code
4 compile your own exe

**Comment by Yakman3** [comment]
If i change this code, would i be able to log in even if i had the mobile authenticator enabled?
The lower prices are great but it won't help if i can't get on, and cookies don't seem to work since steam only gives me 2.

**Comment by immorro** [comment]
@Yakman3 u cant log in with out coockie now, steam authenticator broke it, so...try log in steam through browser - than u receive all coockies

**Comment by Yakman3** [comment]
When logging in through my browser and getting the cookies, I only get steamLogin and sessionid. Where do i get the others?

**Comment by stranger2001** [comment]
круто тут можно компильнуть екзешник со своей фичей и все скачают на изи, изи мани впринципе, даже бот не нужен)

**Comment by immorro** [comment]
@stranger2001 да, можно...но ведь глупо было бы решать проблему, а только после этого выкладывать пофикшеный ехе, не так ли?

@Yakman3 im using chrome extension "awesome cookie manager"

**Comment by mrmikas555** [comment]
@immorro твоя версия, что ты скинул выше, она не работает уже?

**Comment by immorro** [comment]
@mrmikas555 правильно. Работала только на время скидок.
